content/en/docs/getting-started/upgrade.md

```diff
@@ -4,21 +4,26 @@ description: Upgrading Falco on a Linux system
 weight: 3
 ---
 
-This section provides upgrading paths for Falco if previously installed following the [Install](../installation/) section. 
+This section provides upgrading paths for Falco if previously installed following the [Install](../installation/) section.
 
-## Upgrading
-
-According to the installation method you chose, you first have to remove the active kernel module before upgrading Falco to the latest version:
+{{% pageinfo color="warning" %}}
+If you are using the kernel module, please remove it before upgrading Falco to avoid issues during the upgrade.
```

Comment:
Is this still needed?

Reply:
no, it is not strictly required but I would leave it here as a best practice to avoid possible issues